src/ds/bits.h

```diff
@@ -452,5 +452,29 @@ namespace snmalloc
         return (size_t)1 << (m_e + LOW_BITS);
       }
     }
+
+    /**
+     * Implementation of std::min
+     *
+     * std::min is algorithm, so pulls in a lot of unneccessary code
```

Comment:
Probably means 'in `<algorithm>`'. For doc comments, we can use Markdown syntax (Doxygen will pull it out sensibly, so please also stick backticks around `std::min` and `std::max`).